### PR TITLE
add pulp_timeout

### PR DESCRIPTION
**Pull request by maciej-markowski**
#163

this PR adds pulp_timeout parameter.

### CODE REVIEW

File: plugins/doc_fragments/pulp.py [hunk 1]
```diff
@@ -42,6 +42,11 @@ class ModuleDocFragment(object):
       - It is recommended to use this once with the M(pulp.squeezer.status) module at the beginning of the playbook.
     type: bool
     default: false
+  pulp_timeout:
+    description:
+      - timeout value for connecting to pulp api
```
Comment (mdellweg) [inline plugins/doc_fragments/pulp.py]: This is not what timeout does.

Comment (maciej-markowski) [reply plugins/doc_fragments/pulp.py]: `time in seconds to wait for successful connection to pulp api` - would that be a better description?

Comment (mdellweg) [reply plugins/doc_fragments/pulp.py]: No, you are trying to make this parameter available to the not yet rewritten modules:
https://github.com/pulp/squeezer/blob/develop/plugins/doc_fragments/pulp.py#L63

Comment (maciej-markowski) [reply plugins/doc_fragments/pulp.py]: I am not sure if I follow. 
I suppose what you're saying is that you're rewriting modules to use pulp_glue which already has this timeout?
As far as I can see, at least the rpm related modules are still using this openapi.py and it lacks this setting, which causes me some issues that I described in https://github.com/pulp/squeezer/issues/163.
Can you explain what I am supposed to do to get this merged?

Comment (mdellweg) [reply plugins/doc_fragments/pulp.py]: Yes, basically.
You should use the same name and the same description at the very least. So rewriting the modules eventually will not break this again.

**Review by mdellweg — DISMISSED** [review]

**Comment by maciej-markowski** [comment]
I see the pipeline failed due to ansible removing support for python3.10
Can we still merge it and release or do we need to address this issues here?

**Comment by mdellweg** [comment]
Nah, it would be better to fix the ci first.
Would you mind bumping some python versions in the workflows? Please make it a separate PR.

**Comment by maciej-markowski** [comment]
> Nah, it would be better to fix the ci first. Would you mind bumping some python versions in the workflows? Please make it a separate PR.

#166

**Comment by mdellweg** [comment]
Please rebase this.

**Comment by maciej-markowski** [comment]
Are we going to merge https://github.com/pulp/squeezer/pull/166 first?

**Comment by mdellweg** [comment]
I don't think, we need that anymore. I only adds more test combinations. And I am quite happy that we are down to 4 combinations, and still covering all supported ansible versions.

**Review by mdellweg — APPROVED** [review]

**Pull request merged**